cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cluster/DefaultAgentIntervalProvider.java

```diff
@@ -33,6 +34,13 @@ public DefaultAgentIntervalProvider(long interval, long timeout) {
 
     @Override
     public Interval getInterval(Agent agent) {
+        if (agent instanceof AgentIntervalAware) {
+            Long agentInterval = ((AgentIntervalAware) agent).getAgentInterval();
+            if (agentInterval != null && agentInterval > 0) {
+                return new Interval(agentInterval, 2 * agentInterval);
```

Comment:
Please leave a comment here explaining what the `2 *` is for.

Reply:
Commented.